README.md

```diff
@@ -1 +1,20 @@
-# users-manager
\ No newline at end of file
+# users-manager
+
+## Development
+
+```bash
+go install -v google.golang.org/protobuf/cmd/protoc-gen-go@latest
+go install -v google.golang.org/grpc/cmd/protoc-gen-go-grpc@latest
+
+protoc --go_out=. --go-grpc_out=. proto/users.proto
+```
+## Launch
+
+```bash
+docker-compose up --build -d --remove-orphans
```

Comment:
Maybe its better to add this commands to Makefile

Reply:
i don't remember Makefile syntax. It will take some time to implement it. Maybe later